### DISCUSSION TITLE
c++  onnx

### DISCUSSION THREAD

**Discussion by wangzuo-hx** [open-edge-platform/anomalib#520] [Q&A] [answered]
How to use c++(onnxruntime/opencv) to load onnx model and get segmentation results?(e.g. Padim)

**Comment by djdameln** [comment, marked as answer]
Hi, our library does not include any utilities to run models in c++. I would suggest to read the documentation of [ONNX Runtime](https://onnxruntime.ai/docs/) and [OpenVINO](https://docs.openvino.ai/latest/documentation.html) to learn more about how converted models can be deployed in c++.
